can't build tergm/EpiModel with ergm@master

The following error occurs on the mac build:

```
clang -mmacosx-version-min=10.13 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I'/Users/runner/work/_temp/Library/ergm/include' -I/usr/local/include   -fPIC  -Wall -g -O2  -Wall -pedantic -c changestats_operator.c -o changestats_operator.o
In file included from changestats_operator.c:10:
/Users/runner/work/_temp/Library/ergm/include/ergm_util.h:[32](https://github.com/statnet/tergm/actions/runs/4299493639/jobs/7877208639#step:10:33):12: error: implicit declaration of function 'Rprintf' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
  if(name) Rprintf("%s: ", name);
           ^
/Users/runner/work/_temp/Library/ergm/include/ergm_util.h:42:12: error: implicit declaration of function 'Rprintf' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
  if(name) Rprintf("%s:\n", name);
           ^
```

Possibly due to `ergm_util.h` newly calling `Rprintf` but not including the associated header? (6656487d6bc20d8ca77b4ec100e74be9b248ff9f)

--- a/inst/include/ergm_util.h
+++ b/inst/include/ergm_util.h
@@ -10,6 +10,8 @@
 #ifndef _ERGM_UTIL_H_
 #define _ERGM_UTIL_H_
 
+#include <R.h>
+
 /* Calculate the dot product between two vectors. */
 static inline double dotprod(double *x, double *y, unsigned int n){
   double out = 0;

